.github/renovate.json

```diff
@@ -2,8 +2,13 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["github>bitwarden/renovate-config:non-pinned"],
   "separateMajorMinor": true,
-  "enabledManagers": ["cargo", "github-actions", "npm", "nuget"],
+  "enabledManagers": ["cargo", "dockerfile", "github-actions", "npm", "nuget"],
   "packageRules": [
+    {
```

Comment:
⛏️ I prefer these to be alphabetical so it'd go below the Cargo configuration.

Reply:
Me too. Don't know why I missed that.